src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Resolvers/OpenApiConfigurationResolver.cs

```diff
@@ -1,36 +1,46 @@
-using System;
-using System.Linq;
-using System.Reflection;
-
 using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Abstractions;
 using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Configurations;
 using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Extensions;
 using Microsoft.OpenApi.Models;
+using System;
+using System.Linq;
+using System.Reflection;
```

Comment:
Can you please put those lines at the top of the file? Those `System.*` namespaces should go first.